bluebutton-data-pipeline-fhir-load/src/test/java/gov/hhs/cms/bluebutton/datapipeline/fhir/transform/DataTransformerTest.java

```diff
@@ -1501,6 +1503,129 @@ public void transformInsertSNFClaimEvent() {
 	}
 
 	/**
+	 * Verifies that {@link DataTransformer} works correctly when when passed a
+	 * single {@link HospiceClaimGroup} {@link RecordAction#INSERT}
+	 * {@link RifRecordEvent}.
+	 */
+	@SuppressWarnings({ "rawtypes", "unchecked" })
+	@Test
+	public void transformInsertHospiceClaimEvent() {
+		// Create the mock bene to test against.
+		HospiceClaimGroup record = new HospiceClaimGroup();
+		record.version = RifFilesProcessor.RECORD_FORMAT_VERSION;
+		record.recordAction = RecordAction.INSERT;
+		record.beneficiaryId = "42";
+		record.claimId = "SuttersMill";

```

Comment:
Oh... there goes my (not very funny) joke. https://en.wikipedia.org/wiki/Sutter%27s_Mill, a goldrush "claim".

Bah humbug, I say!